source/guides/guides/parallelization.md

```diff
@@ -2,45 +2,74 @@
 title: Parallelization
 ---
 
-{% note warning %}
-This documentation is still a work in progress.
+{% note info %}
+
+# {% fa fa-graduation-cap %} What You'll Learn
+
+- How to parallelize test runs
+- How to group test runs
+- Strategies for grouping test runs
+- How test spec load-balancing works
+- Gaining better insights into tests with Dashboard spec views
+
 {% endnote %}
 
 # Overview
 
-If your project has a large number of tests, it can take a long time for tests to complete running when run serially on one machine. Spreading your tests across many virtual machines and running tests in parallel can save your team time and money when running tests in Continuous Integration.
+If your project has a large number of tests, it can take a long time for tests to complete running when run serially on one machine. Spreading your tests across many virtual machines and running tests in parallel can save your team time and money when running tests in Continuous Integration (CI).
```

Comment:
"complete running when run serially on one machine" -> "complete running serially on one machine"
 "Spreading your tests across many virtual machines and running tests in parallel" ->  "Running tests in parallel across many virtual machines"